pkg/instrumentation/categories.go

```diff
@@ -66,6 +67,11 @@ func determineCategoryFromArgs(args []string, knownCommands []string, flagsAllow
 			if slices.Contains(flagsAllowList, flagName) {
 				flags = append(flags, strings.ToLower(flagName))
 			}
+
+			// a shorthand variant for debug
+			if flagName == configuration.DEBUG_FLAG_SHORTHAND {
```

Comment:
Suggestion: While this solution solves the missing support for the `-d` short hand flag, there is a general missing support for shorthand flags and I think we can adapt the implementation slightly to make this work. Happy to hop on a call for this, otherwise I can push a commit here with my proposal.

Reply:
@PeterSchafer feel free to push your proposal into this branch :)

Reply:
I opted for this lightweight approach as this is the only shorthand flag we are using at the moment.

Reply:
Makes sense! But I think there are actually more shorthand flags in the wild than we expect.

Reply:
I just pushed a minor change that moves the shorthand flag out of the configuration package, since the configuration package is meant to hold keys that are used to access the configuration. a shorthand flag is a cmd specific thing though.